legend-pure-ide-light/src/main/java/org/finos/legend/pure/ide/light/api/RuntimeOptions.java

```diff
@@ -0,0 +1,32 @@
+package org.finos.legend.pure.ide.light.api;
+
+import io.swagger.annotations.Api;
+import org.finos.legend.pure.ide.light.session.PureSession;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+
+public class RuntimeOptions
+{
+    private PureSession pureSession;
+
+    public RuntimeOptions(PureSession pureSession)
+    {
+        this.pureSession = pureSession;
+    }
+
+    @GET
+    @Path("setOption/{name}/{value}")
```

Comment:
From UI perspective, we should have an endpoint that could return the entire map so that we can populate the initial states in the web application